etcdmgmt/etcd-mgmt.go

```diff
@@ -0,0 +1,60 @@
+package etcdmgmt
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func StartEtcd() error {
+
+	log.Info("Starting etcd")
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Could not able to get hostname")
+		return err
+	}
+
+	HostPort2379 := "http://" + hostname + ":2379"
+
+	HostPort2380 := "http://" + hostname + ":2380"
+
+	EtcdStart := exec.Command("/bin/etcd",
+		"-listen-client-urls", HostPort2379,
+		"-advertise-client-urls", HostPort2379,
+		"-listen-peer-urls", HostPort2380,
+		"-initial-advertise-peer-urls", HostPort2380,
+		"--initial-cluster", "default="+HostPort2380)
+
+	err = EtcdStart.Start()
+	if err != nil {
+		log.Fatal("Could not start etcd daemon.")
+		return err
+	}
+
+	//Checking health of etcd cluster
```

Comment:
Can we really wait for the cluster to be healthy for indefinite time? I think we should error out after a good amount of retries.